src/screens/WebsocketResearchScreen/SampleChatScreen.tsx

```diff
@@ -55,7 +55,8 @@ const SampleChatScreen = () => {
     goBackFromChatScreen,
     goToChatInfoScreen,
     sendChat,
-    updateChatContinuity
+    updateChatContinuity,
+    loadingChat
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The new variable `loadingChat` is destructured from the `useChatScreenHook` but it's not used anywhere in the provided code. If it's not used elsewhere in the component, it should be removed to avoid unnecessary variable declarations.
```diff
-    loadingChat
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->